javascript/src/main/pages/Admin/ViewLogins.js

```diff
@@ -13,6 +29,7 @@ const ViewLogins = () => {
     <>
       <h2 className="text-left">Login History</h2>
       <LoginTable loginTable={loginTable || []} admin={true}/>
+      <Button><CSVLink style={{color: "white"}} headers={headers} data={loginTable || []} filename = {"LoginTable.csv"}>Download CSV</CSVLink></Button>
```

Comment:
When the login table is very long, it would be nice to see the download button above the table. Would you be able to place the download button above the table? Otherwise, it looks good.